mkdocs.yml

```diff
@@ -0,0 +1,227 @@
+# Project
+site_name: Synapse Python/Command Line Client Documentation
+site_author: Bryan Fauble
+site_description: >-
+  A Python client for Sage Bionetworks' Synapse, a collaborative, open-source research
+  platform that allows teams to share data, track analyses, and collaborate. The Python
+  client can be used as a library for development of software that communicates with
+  Synapse or as a command-line utility.
+
+# Repository
+repo_url: https://github.com/Sage-Bionetworks/synapsePythonClient
+repo_name: synapsePythonClient
+
+# Navigation
+nav:
```

Comment:
This nav section is the biggest thing to be concerned about for this ticket. This is everything for the first round of documentation that I thought we should include. This is based off the documentation from https://diataxis.fr/

Reply:
What do you think about adding [`navigation.tabs` and/or `navigation.tabs.sticky`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-tabs) feature?  Because the Python client docs is pretty big, I think adding tabs could be useful / more organized.

For some references, both [mkdocstrings](https://mkdocstrings.github.io/) and [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/) use it for their docs site.

Reply:
I like navigation.tabs!

I'm thinking without sticky so that more of the page is viewable in longer articles.


Without sticky:
![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/3bd85fae-e4b8-4003-99c8-2cb4899b2a9e)


With sticky:
![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/f03739c1-70d1-453d-8907-bda9bc4fa6ed)